modules/core/src/main/java/org/apache/ignite/internal/visor/cache/VisorCacheMetrics.java

```diff
@@ -456,6 +465,20 @@ public int getKeySize() {
         return keySize;
     }
 
+    /**
+     * @return Number of non-{@code null} values in the cache.
```

Comment:
Maybe to add to comment "as a long type"?

Reply:
Done.